runners/extensions-java/metrics/src/main/java/org/apache/beam/runners/extensions/metrics/MetricsHttpSink.java

```diff
@@ -86,8 +93,8 @@ public MetricNameSerializer(Class<MetricName> t) {
     public void serialize(MetricName value, JsonGenerator gen, SerializerProvider provider)
         throws IOException {
       gen.writeStartObject();
-      gen.writeObjectField("name", value.name());
-      gen.writeObjectField("namespace", value.namespace());
```

Comment:
yes I replaced them with get* versions to simplify serialization code but they were re-introduced to be compatible with dataflow hosted engine which uses .name and .namespace which I could not know of because it is not opensource. Anyway good catch !  Dataflow is supposed to have migrated to .get* versions now, can you check ? If so we can remove the deprecated versions from the api.

Reply:
Got it, I was just doing some code cleaning as I went along. I've verified that removing the deprecated methods doesn't break anything and added that to this PR. I'll update the commit message as well.